atef/widgets/config/utils.py

```diff
@@ -1159,7 +1160,17 @@ def data(self, index: QtCore.QModelIndex, role: int) -> Any:
 
         return None
 
+    def data_updated(self) -> None:
+        """ method for calling dataChanged on the entire view """
+        top_left = self.index(0, 0, QtCore.QModelIndex())
+        bottom_right = self.index(self.rowCount(QtCore.QModelIndex()),
+                                  self.columnCount(QtCore.QModelIndex()),
+                                  QtCore.QModelIndex())
+        self.dataChanged.emit(top_left, bottom_right)
 
+
+# TODO: Rename this and related helpers to be more specific
```

Comment:
TODO reminder; going to make an issue I assume?

Reply:
Yeah I intend to address this and other TODOs when I rework widget / tree loading